textproc/moor/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/walles/moor 2.0.4 v
+go.offline_build    no
+revision            0
+
+description         Moor is a pager. It's designed to just do the right thing \
+                    without any configuration.
+
+long_description    Moor should work as a drop-in replacement for Less. \
+                    It features syntax highlighting, ANSI color coded text \
+                    support, incremental search and automatic decompression, \
+                    among others.
+
+checksums           rmd160  29fb69fbf98c61b14dde86f49ebf88c5c86aa730 \
+                    sha256  7d8c9c5da7e2b95cb0082ddb269da978247f5bbb62b1f2556760046b70c1051b \
+                    size    3013176
+
+categories          textproc
+installs_libs       no
+license             FreeBSD
+maintainers         {gmail.com:j.lopez.r @jlopezr} \
+                    {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+build.args-append   ./cmd/${name}
+build.pre_args-append \
+    -ldflags \"-X main.versionString=${version}\"
+
+destroot {
+    xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
+
+    xinstall -d ${destroot}${prefix}/share/man/man1
```

Comment:
You don't need to create standard directories like this one; MacPorts has already created it for you.
```suggestion
```